grafana-plugin/src/components/ScheduleUserDetails/ScheduleUserDetails.module.css

```diff
@@ -37,21 +37,41 @@
   color: #ccccdc;
 }
 
+.username {
+  word-break: break-all;
+}
+
 .timezone-wrapper {
   display: flex;
+  width: 92%;
 }
 
 .timezone-icon {
-  width: 10%;
+  width: 8%;
+  margin-right: 8px;
```

Comment:
Can't we just use `flex-grow: 1` on `.timezone-wrapper` so that it takes as much space it needs instead of hardcoding `92%` and `8%`? Just asking :thinking:

Reply:
You are right, Fixed that